service/source.go

```diff
@@ -448,3 +468,20 @@ func (g *AutoParameterGenerator) enum(index []int, typ reflect.Type, f func(inde
 	}
 	return nil
 }
+
+func zero(t reflect.Type) interface{} {
+	switch t.Kind() {
+	case reflect.Bool:
+		return false
+	case reflect.Uint, reflect.Uint8, reflect.Uint16, reflect.Uint32, reflect.Uint64:
+		return 0
```

Comment:
It's wrong. You can't treat `int` as `uint`, `uint8` and etc. 
All values should have its own type rather `int`.